Source/WebCore/platform/ThreadSafeDataBuffer.h

```diff
@@ -56,7 +66,14 @@ friend class ThreadSafeDataBuffer;
 };
 
 class ThreadSafeDataBuffer {
+private:
+    friend struct IPC::ArgumentCoder<ThreadSafeDataBuffer, void>;
 public:
+    static ThreadSafeDataBuffer create(RefPtr<ThreadSafeDataBufferImpl>&& impl)
```

Comment:
This shouldn't be public if it's only used by the ArgumentCoder, which we have befriended.